### DISCUSSION TITLE
Examples with fundamental concepts

### DISCUSSION THREAD

**Discussion by activivan** [speedata/publisher#583] [Q&A] [answered]
Hi there, I have difficulties with the documentation and can not understand why certain things behave the way they do. Are there any examples that underline the fundamentals?
Also, what is the best way to get support? I couldn't find a Discord server or online community except this, is this the right place to ask for support / to ask "dumb" or obvious questions? Regards!

**Comment by pgundlach** [comment, marked as answer]
There is the examples repository, have you seen it? https://github.com/speedata/examples/

Actually, the best way to get support is this discussion page. There is a [matrix channel](https://matrix.to/#/#speedata_publisher:gitter.im) for support as well, but I am only available during office hours...  and I don't get any unread messages via mail, so I constantly miss unread messages. But I will keep an eye on it.

There are usually no dumb questions, this is mostly a sign of lack of documentation.
